src/app/[lng]/layout.tsx

```diff
@@ -33,6 +32,7 @@ export default async function RootLayout({
       lang={lng || fallbackLng}
       suppressHydrationWarning
       className={fonts.map((f) => f.variable).join(' ')}
+      style={{ scrollBehavior: 'smooth' }}
```

Comment:
@oktapodia Do you have a better solution than that?

Reply:
You can use the GlobalStyles but I think it's ok